phpBB/ext/phpbb/teamsecurity/composer.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "phpbb/teamsecurity",
```

Comment:
The one issue there is what's being committed here has `website[at]phpbb.com` hardcoded as an email recipient. I updated this extension since then to instead use `$this->config['board_contact'] and $this->config['board_contact_name']` so it's not hardcoded. I guess it could be changed yet again so that the email recipient is set in this extensions configs too.